itou/siaes/models.py

```diff
@@ -169,9 +186,16 @@ def has_member(self, user):
             siaemembership__user=user, siaemembership__user__is_active=True
         ).exists()
 
+    def has_admin(self, user):
+        return self.members.filter(
+            siaemembership__user=user,
+            siaemembership__user__is_active=True,
+            siaemembership__is_siae_admin=True,
+        ).exists()
+
     @property
     def siren(self):
-        # pylint: disable=E1136
+        # pylint: disable=unsubscriptable-object
```

Comment:
Je ne comprends pas trop pourquoi Pylint souligne ici une erreur possible. Il est pourtant possible d'itérer sur `siret`, c'est une chaîne. Tu as une idée ?

Reply:
C'est un faux positif, il y en a plein leur issues https://github.com/PyCQA/pylint/search?q=unsubscriptable&type=Issues